AndroidSDK/src/com/leanplum/LeanplumPushService.java

```diff
@@ -881,4 +903,100 @@ private static boolean hasAppIDChanged(String currentAppId) {
     }
     return false;
   }
+
+  /**
+   * Set default notification channel.
+   *
+   * @param context The application context.
+   * @param defaultChannelId The id of the channel.
+   */
+  public static void setDefaultNotificationChannel(Context context, String defaultChannelId) {
```

Comment:
I would remove all the following methods and leave this up to the developer.
In my opinion we should only care about reading notification channels. Adding this functionality will add possibility of overhead, increase size and maybe even issues in the future...
We could leave this in separate branch, in case someone asks for it...  I would not include it in SDK, maybe we can distribute separate helper class for this...